tabgpt/data/house_prices/data_setup.py

```diff
@@ -49,31 +50,32 @@ def setup(self, all_features=True):
         df_train_full, numerical_features, categorical_features = self.read(all_features)
        
         df_train, df_val = train_test_split(df_train_full, test_size=0.2, random_state=666)
-        num_max = df_train[numerical_features].abs().max()
-        df_train[numerical_features] = df_train[numerical_features] / num_max
-        df_val[numerical_features] = df_val[numerical_features] / num_max
-        df_train["target"] = np.log(1 + df_train["SalePrice"])
-        df_val["target"] = df_val["SalePrice"]
+       
+        self.setup_scaler(numerical_features)
```

Comment:
You now have to `setup_scaler` , `scale_columns` and `set_target_column` within the setup method